test/functional/mempool_expiry.py

```diff
@@ -42,7 +42,8 @@ def test_transaction_expiry(self, timeout):
 
         # Add prioritisation to this transaction to check that it persists after the expiry
         node.prioritisetransaction(parent_txid, 0, COIN)
-        assert_equal(node.getprioritisedtransactions()[parent_txid], { "fee_delta" : COIN, "in_mempool" : True})
+        # Fee rate found in test/functional/test_framework/wallet.py create_self_transfer
+        assert_equal(node.getprioritisedtransactions()[parent_txid], { "fee_delta" : COIN, "in_mempool" : True, "modified_fee": COIN + COIN * (0.003 * 104 / 1000) })
```

Comment:
Added a comment ontop showing where I got these numbers from

Reply:
These are still [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)), even if they are accurate. The comment is helpful, but doesn't change the fact that this test relies on the implementation of `create_self_transfer`, making it brittle. If we were to change the implementation so that the virtual size could be something other than 104, or if we adjusted the default feerate to something other than 0.003, this test would break, even though the mempool/RPC logic hasn't changed.

I was suggesting that you assign variables and reuse them in this test:
- `fee_delta` to use in `prioritisetransaction`
- `parent_fee` meaning the absolute fees of the tx. If you're having trouble getting the fee, it'll be one of the results returned from `send_self_transfer`.
- assert here that `modified_fee` is equal to `parent_fee + fee_delta`

Reply:
not sure why I didn't think of just using the value returned from `create_self_transfer` fixed in [24b490f](https://github.com/bitcoin/bitcoin/pull/28885/commits/24b490f292149e6151dcddaf11ac0d5bf16dde44)